assets/javascripts/discourse/components/modal/diff-modal.gjs

```diff
@@ -45,11 +45,16 @@ export default class ModalDiffModal extends Component {
 
     // Prevents flash by showing the
     // original text when the diff is empty
-    return this.selectedText;
+    return this.escapedSelectedText;
   }
 
   get isStreaming() {
-    return this.diffStreamer.isStreaming || this.smoothStreamer.isStreaming;
+    // diffStreamer stops "streaming" when it is finished with a chunk
+    return (
+      this.diffStreamer.isStreaming ||
+      !this.diffStreamer.isDone ||
```

Comment:
I feel like that when you are done you should not be streaming anymore. So setting isDone to true should also set isStreaming to false, otherwise we will plague the code with this double check.